packages/client/src/index.ts

```diff
@@ -44,3 +44,5 @@ export type { NOOP } from "./util/NOOP.js";
 
 export { createMinimalClient } from "./createMinimalClient.js";
 export type { MinimalClient } from "./MinimalClientContext.js";
+
+export { augment } from "./object/fetchPage.js";
```

Comment:
I'm not totally thrilled with this becoming part of the public API, but appreciate that its super hard to get the typings to work out. Perhaps this is covered by the 'make hard things possible' side of things instead of the 'make most things easy' part.

Reply:
It definitely works without doing this. This is only needed if you want the typesafety on picking properties.

Reply:
Lets leave this for now